spec/Machine/init.js

```diff
@@ -26,7 +26,28 @@ module.exports = (sampleFile) => {
     },
 
     getResultWithLineControl: () => {
-      return fs.readFileSync(sampleFile + '.out-lc', 'utf-8');
+      let content;
+      try {
```

Comment:
Main thing is path sep, not platform.  In my opinion, needless to handle many types of fs, if it has same filepath separator.